next-app/src/lib/PageHeader.svelte

```diff
@@ -1,3 +1,3 @@
-<h1 class='text-2xl md:text-3xl'>
+<h1 class='text-2xl md:text-3xl { $$props.class }'>
```

Comment:
Instead of documenting this everywhere it occurs, what would think of this approach:  [0956b5c](https://github.com/sillsdev/web-languageforge/pull/1416/commits/0956b5ca30740378b9bf9d10e3d8605037c18454)

Reply:
yes, this is fine.  Thank you!